### DISCUSSION TITLE
INSERT - RETURNING MINIMAL

### DISCUSSION THREAD

**Discussion by PedroLoboA** [supabase/supabase#1212] [Questions] [answered]
One question:

I want to insert a data into a table, but because the way i built my SELECT policy i´am not able to get the returning result from the insert. 

If i do { returning: 'minimal' } the insert works fine but i am not able to subscribe(),

to be able to insert and get the returning correctly i would need to insert into two tables and then get the return, thats someway to do that?

**Comment by kiwicopple** [comment, marked as answer]
> i am not able to subscribe()

If the problem is the subscription, can I check that you have realtime enabled for this table? https://supabase.io/docs/guides/api#managing-realtime

> to be able to insert and get the returning correctly i would need to insert into two tables and then get the return

You should be able to do it with the proper policies in place. Can you share some code and we can help to determine what changes can be made?

**Reply by PedroLoboA** [reply]
My bad, i through the real time was enable but wasnt.

Thank you for the fast answer!

**Reply by KingMCMS** [reply]
Was geht
